Public/Src/Cache/VerticalStore/Deployment.dsc

```diff
@@ -4,34 +4,22 @@
 import * as Deployment from "Sdk.Deployment";
 import * as Managed from "Sdk.Managed";
 
-namespace Deployment{
-
-
-export declare const qualifier: BuildXLSdk.DefaultQualifier;
-
-@@public
-export const deployment: Deployment.Definition = {
-    contents: BuildXLSdk.isDotNetCoreBuild ?
-    [
-        BasicFilesystem.dll,
-        BuildCacheAdapter.dll,
-        InMemory.dll,
-        Interfaces.dll,
-        ImplementationSupport.dll,
-        MemoizationStoreAdapter.dll,
-        VerticalAggregator.dll,
-    ]:[
-        Analyzer.exe,
-        BasicFilesystem.dll,
-        BuildCacheAdapter.dll,
-        Compositing.dll,
-        InMemory.dll,
-        InputListFilter.dll,
-        Interfaces.dll,
-        ImplementationSupport.dll,
-        MemoizationStoreAdapter.dll,
-        VerticalAggregator.dll,
-    ]
-};
-
+namespace Deployment {
+    @@public
+    export const deployment: Deployment.Definition = {
+        contents: [
+            InMemory.dll,
+            Interfaces.dll,
+            BasicFilesystem.dll,
+            BuildCacheAdapter.dll,

```

Comment:
You might want to check with David & Lance: They were checking if we could remove these..

Reply:
This is semantically still the same, just without duplication. I'll let them decide once they refactor.